blocks/title-banner/title-banner.js

```diff
@@ -1,16 +1,23 @@
-import { getMetadata } from '../../scripts/aem.js';
+import { fetchPlaceholders, getMetadata, toCamelCase } from '../../scripts/aem.js';
 import { h1, strong } from '../../scripts/dom-builder.js';
 
 export default async function decorate(block) {
-  if (block.classList.contains('author')) {
-    // if the block is empty, add a title
-    if (!block.querySelector('h1')) {
+  if (!block.querySelector('h1')) {
+    let dynamicHeadline = '';
+    if (block.classList.contains('author')) {
       const author = getMetadata('author');
       if (author) {
-        const headline = h1('Articles by ');
-        headline.appendChild(strong(author));
-        block.prepend(headline);
+        dynamicHeadline = h1('Articles by ', strong(author));
       }
+    } else if ((block.classList.contains('tags') && window.location.pathname.includes('/tags/')) || (block.classList.contains('topics') && window.location.pathname.includes('/topics/'))) {
```

Comment:
Is there a reason why authors need to specify `tag/topic` qualifier on the block or can it be just derived from the path thus simplifying a bit for authors?

Reply:
This is mostly for the styles. The banner is used across multiple page types. Some we can derive from todays URLs (which might change) others not.
Also have in mind these pages are mostly based on a template. Author does not create pages for each tag or topic. Most likely these are create once and never touched pages.

Reply:
> Some we can derive from todays URLs (which might change) others not.
ok, in that case it does make sense to make it explicit and anyway as you conveyed, these pages aren't often authored.